docs/core/preview3/tools/dotnet-pack.md

```diff
@@ -75,6 +69,10 @@ Configuration to use when building the project. If not specified, will default t
 
 Updates the star in `-*` package version suffix with a specified string.
 
+`--verbosity <LEVEL>`
+
+Set the verbosity level of the command. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
```

Comment:
I like how you describe them here (I'd just change to start with Sets). We could try to standardize on other topics. We could either use an include file for this and other common options such as --help so they all look the same. Let's discuss it tomorrow.